lib/hanami/action/mime.rb

```diff
@@ -16,6 +16,12 @@ module Mime
       # @api private
       HTTP_ACCEPT          = 'HTTP_ACCEPT'.freeze
 
+      # The key that returns content mime type from the Rack env
+      #
+      # @since 1.1.1
```

Comment:
Not sure about that 🤔

Reply:
You can write `x.x.x` instead, and the release process will replace it with the correct version :)